incremental/ubuntu18-upgrade-to-stein.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2019, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -evu
+
+source lib/functions.sh
+source lib/vars.sh
+
+require_ubuntu_version 18
+
+export OSA_SHA="stable/stein"
+export SKIP_INSTALL=${SKIP_INSTALL:-'no'}
+export RPC_PRODUCT_RELEASE="stein"
+export RPC_ANSIBLE_VERSION="2.7.8"
```

Comment:
OSA bumped to 2.7.9

Reply:
Did they?  I pulled the current version # from here: https://github.com/openstack/openstack-ansible-tests/blob/stable/stein/test-ansible-deps.txt#L11

Reply:
Yes https://github.com/openstack/openstack-ansible/blob/stable/stein/scripts/bootstrap-ansible.sh#L25

So they test something different they install, fantastic

Reply:
Guess it's a mismatch, https://github.com/openstack/openstack-ansible/blob/stable/stein/scripts/bootstrap-ansible.sh#L25 heh, k will update and merge.